wicket-util/pom.xml

```diff
@@ -47,22 +47,6 @@
 		<dependency>
 			<groupId>org.junit.jupiter</groupId>
```

Comment:
This dependency is global (set in parent pom), maybe it can be dropped here?

Reply:
Yes, thanks. Code is removed.